src/sentry/integrations/api/endpoints/organization_coding_agents.py

```diff
@@ -0,0 +1,84 @@
+from __future__ import annotations
+
+import logging
+
+from rest_framework.request import Request
+from rest_framework.response import Response
+
+from sentry import features
+from sentry.api.api_owners import ApiOwner
+from sentry.api.api_publish_status import ApiPublishStatus
+from sentry.api.base import region_silo_endpoint
+from sentry.api.bases.organization import OrganizationEndpoint
+from sentry.constants import ObjectStatus
+from sentry.integrations.coding_agent.utils import get_coding_agent_providers
+from sentry.integrations.services.integration import integration_service
+
+logger = logging.getLogger(__name__)
+
+
+@region_silo_endpoint
```

Comment:
should integrations be in region silo or control silo?

Reply:
my understanding is region following https://github.com/getsentry/sentry/blob/592f7f9ad3ada3325227ca97fe1e005e6ad3b778/src/sentry/integrations/api/endpoints/organization_integration_repos.py#L26-L27